src/main/java/teammates/sqllogic/api/SqlEmailGenerator.java

```diff
@@ -1001,6 +1001,32 @@ public EmailWrapper generateNewAccountRequestAdminAlertEmail(AccountRequest acco
         return email;
     }
 
+    /**
+     * Generates the acknowledgement email to be sent to the person who submitted {@code accountRequest}.
+     */
+    public EmailWrapper generateNewAccountRequestAcknowledgementEmail(AccountRequest accountRequest) {
+        String name = accountRequest.getName();
+        String institute = accountRequest.getInstitute();
+        String emailAddress = accountRequest.getEmail();
+        String comments = accountRequest.getComments();
+        if (comments == null) {
+            comments = "";
+        }
+        String[] templateKeyValuePairs = new String[] {
+                "${name}", name,
+                "${institute}", institute,
+                "${emailAddress}", emailAddress,
+                "${comments}", comments,
+        };
+        String content = Templates.populateTemplate(
+                EmailTemplates.INSTRUCTOR_NEW_ACCOUNT_REQUEST_ACKNOWLEDGEMENT, templateKeyValuePairs);
+        EmailWrapper email = getEmptyEmailAddressedToEmail(Config.SUPPORT_EMAIL);
```

Comment:
You may have forgotten to change this. The email should be sent to the instructor's email address, not to the TEAMMATES email address.